dev_tools/conf/apt-list-dev-tools.txt

```diff
@@ -1 +1,2 @@
+protobuf-compiler
```

Comment:
Try this on a non-google-workstation machine and confirm that it works before adding it as a dependency. In my testing, it installs a too-old version of the protobuf compiler on external machines.

Reply:
Um, not sure what you'd do if you figured this out.  This is developer install instructions I think dealing with bad install because you have some strange distribution setup is out of scope of what we need to handle.

Reply:
I removed it since apparently if you install this outside of ubuntu it can end up giving you the wrong version.  Instead I've added it to the developer isntructions and in the python 2 generating script it yells a warning at you if you don't have protoc installed and directs you to the github page describing how to install.